addons/sale/models/sale_order.py

```diff
@@ -146,6 +146,11 @@ class SaleOrder(models.Model):
         store=True, readonly=False, required=True, precompute=True,
         states=LOCKED_FIELD_STATES,
         domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]",)
+    # Credit Limit related fields
+    partner_credit = fields.Monetary(related='partner_id.commercial_partner_id.credit')
+    partner_credit_limit = fields.Monetary(related='partner_id.credit_limit_compute')
```

Comment:
why is the credit taken from the partner but the credit_limit from the partner commercial_partner ?

Reply:
If I'm correct, it does not matter whether you use a contact of a company or the company itself, when invoicing.
The invoice will automatically be posted in the company itself, not the contact.
At least that's my understanding :wink: 
But it might be worth explicitly using the `commerical_partner_id` on both fields for clarity's sake ?

Reply:
My knowledge of the differentiation & logic of partner & commercial partners is a bit shallow (at least from a func POV), but it's the kind of thing that seems technically confusing at first glance, so either a little explanatory comment or use the commercial_partner_id for both fields related would be nice imho :).

Reply:
Definitely agree on this point, even on a functional point of vue the commercial partner makes more sense here.

Reply:
Can you set a credit limit on contacts or only on the commercial partner? You should not be able to set credit limits on them if you are never looking at them. (and vice versa)

Reply:
> Can you set a credit limit on contacts or only on the commercial partner?
No you can't. If you go to the Invoicing/Accounting tab on a (contact) partner, it is empty, except for a link that redirects you to the commercial partner 😉